src/services/mongoose/wallet-invoices.ts

```diff
@@ -161,3 +139,11 @@ export const WalletInvoicesRepository = (): IWalletInvoicesRepository => {
     deleteUnpaidOlderThan,
   }
 }
+
+const walletInvoiceFromRaw = (result): WalletInvoice => ({
```

Comment:
Need to define `InvoiceUserRecord` and type this correctly.

Reply:
should be done, but not really the goal of this PR